archive/docs/Installing-the-project.md

```diff
@@ -0,0 +1,31 @@
+[Download the jar from Maven](http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22io.appium%22%20AND%20a%3A%22java-client%22) or add the following to pom.xml:
+
+```
+<dependency>
+  <groupId>io.appium</groupId>
+  <artifactId>java-client</artifactId>
+  <version>4.1.2</version>
+</dependency>
+```
+
+It currently depends on selenium-java 2.53.1. If it is necessary to use another version of Selenium then you can configure pom.xml as follows:
```

Comment:
@SrinivasanTarget I think it is not necessary to write about the certain version which java client depends on. The fact that the client depends on Selenium java is much more important.